app/pages/prescription/ScheduleForm.js

```diff
@@ -74,23 +84,27 @@ const ScheduleForm = props => {
             innerRef={refs[index]}
             id={`${fieldArrayName}.${index}.start`}
             name={`${fieldArrayName}.${index}.start`}
-            error={getFieldError(fieldArrayMeta, index, 'start')}
+            error={getFieldError(`${fieldArrayName}.${index}.start`, useFormikContext())}
```

Comment:
The use of a hook here will cause issues when the number of schedules changes since the set of hooks executed per render cycle will change. I see that you're calling the hook pretty often to pass to `getFieldError` - is there an issue with calling the hook to get the context once at the top of the render and then just passing a reference to it here (and elsewhere) as opposed to calling the hook each time? 

The only other solution I've found for this is to break out the component that needs the hook call into its own component.

Reply:
Good catch.  No issue with simply passing a common reference.